tests/test_utils.py

```diff
@@ -117,6 +117,17 @@ def test_merge_date_ranges(self):
         self.assertEqual(len(ranges), 1)
         self.assertEqual(ranges[0], (datetime.datetime(2000, 1, 1), datetime.datetime(2100, 1, 1)))
 
+        # Case 10
+        dates = [(datetime.datetime(2006, 1, 1), datetime.datetime(2008, 1, 1)),
+                 (datetime.datetime(1999, 1, 1), datetime.datetime(2000, 1, 1)),
+                 (datetime.datetime(1900, 1, 1), datetime.datetime(2100, 1, 1))]
+
+        ranges = [r for r in merge_date_ranges(dates)]
+        print(ranges)
```

Comment:
Please remove this `print`.

Reply:
Done. Sorry for not removing it earlier.

Thanks!